src/components/QuickOpenModal.js

```diff
@@ -358,11 +370,12 @@ export class QuickOpenModal extends Component<Props, State> {
 
   render() {
     const { enabled, query } = this.props;
-    const { selectedIndex, results } = this.state;
+    let { selectedIndex, results, isLoading } = this.state;
```

Comment:
`const` -> `let` am I missing something here?

Reply:
agreeing with @AnshulMalik -- since none of these variables are reassigned (and since they are part of state and should only be updated via setState), we can use const here!

Reply:
I'll fix this right away